gulpfile.js

```diff
@@ -63,16 +63,15 @@ const WEBAPP_OUT = 'www';
 
 // Copies dependencies imported by [cordova/electron]_index.html.
 function copyIndexDependencies() {
-  const babelPolyfill = 'node_modules/babel-polyfill/dist/polyfill.min.js';
-  const webcomponentsEs5 = 'node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js';
-  const webcomponentsLoader =
-      'node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js';
-  const webanimations = 'node_modules/web-animations-js/web-animations-next-lite.min.js';
-
-  runCommand(`cp -v ${babelPolyfill} ${WEBAPP_OUT}/babel-polyfill.min.js`);
-  runCommand(`cp -v ${webcomponentsEs5} ${WEBAPP_OUT}/webcomponents-loader.js`);
-  runCommand(`cp -v ${webcomponentsLoader} ${WEBAPP_OUT}/custom-elements-es5-adapter.js`);
-  return runCommand(`cp -v ${webanimations} ${WEBAPP_OUT}/web-animations-next-lite.min.js`);
+  runCommand(`cp -v 'node_modules/babel-polyfill/dist/polyfill.min.js' ${
+      WEBAPP_OUT}/babel-polyfill.min.js`);
+  runCommand(`cp -v 'node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js' ${
```

Comment:
We're already copying these files in `src/www/build_action.sh`, so we can remove them. Also, we want to return the last `runCommand` so the caller can wait on it.